allennlp/models/encoder_decoders/wikitables/wikitables_semantic_parser.py

```diff
@@ -570,7 +655,9 @@ def _action_history_match(predicted: List[int], targets: torch.LongTensor) -> in
     @overrides
     def get_metrics(self, reset: bool = False) -> Dict[str, float]:
         return {
-                'parse_acc': self._action_sequence_accuracy.get_metric(reset)
+                'dpd_acc': self._action_sequence_accuracy.get_metric(reset),
```

Comment:
Can you add a quick docstring here? These are complicated metrics and it's not clear what they represent.

Reply:
Done.